docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -0,0 +1,1673 @@
+
+**Syntax:**
+
+
+.. storm-cli:: graph --help
+
+
+.. _storm-iden:
+
+iden
+----
+
+The ``iden`` command lifts one or more nodes by their node identifier (node ID / iden).
+
+**Syntax:**
+
+
+.. storm-cli:: iden --help
+
+
+**Example:**
+
+- Lift the node with node ID 20153b758f9d5eaaa38e4f4a65c36da797c3e59e549620fa7c4895e1a920991f:
+
+
+.. storm-pre:: [inet:ipv4=1.2.3.4]
+.. storm-cli:: iden 20153b758f9d5eaaa38e4f4a65c36da797c3e59e549620fa7c4895e1a920991f
+
+
+**Usage Notes:**
+
+- The node ID (iden) for a given node can be obtained by lifting the node using the ``--debug`` or ``--raw`` option to the :ref:`syn-storm` command:
+
+  - ``storm --debug inet:fqdn=woot.com``
+
+.. _storm-layer:
+
+layer
+-----
+
+Storm includes ``layer.*`` commands that allow you to work with layers (see :ref:`gloss-layer`).
+
+- `layer.add`_
+- `layer.set`_
+- `layer.get`_
+- `layer.list`_
+- `layer.del`_
+
+Help for individual ``layer.*`` commands can be displayed using:
+
+  ``storm <command> --help | -h``
+
+.. _storm-layer-add:
+
+layer.add
++++++++++
+
+The ``layer.add`` command adds a layer to the Cortex.
+
+**Syntax**
+
+
+.. storm-cli:: layer.add --help
+
+
+.. _storm-layer-set:
+
+layer.set
++++++++++
+
+The ``layer.set`` command sets an option for the specified layer.
+
+**Syntax**
+
+
+.. storm-cli:: layer.set --help
+
+.. _storm-layer-get:
+
+layer.get
++++++++++
+
+The ``layer.get`` command retrieves the specified layer from a Cortex.
+
+**Syntax**
+
+
+.. storm-cli:: layer.get --help
+
+
+.. _storm-layer-list:
+
+layer.list
+++++++++++
+
+The ``layer.list`` command lists the available layers in a Cortex.
+
+**Syntax**
+
+
+.. storm-cli:: layer.list --help
+
+
+.. _storm-layer-del:
+
+layer.del
++++++++++
+
+The ``layer.del`` command deletes a layer from a Cortex.
+
+**Syntax**
+
+
+.. storm-cli:: layer.del --help
+
+.. _storm-lift:
+
+lift
+----
+
+Storm includes ``lift.*`` commands that allow you to perform specialized lift operations.
+
+- `lift.byverb`_
+
+Help for individual ``lift.*`` commands can be displayed using:
+
+  ``storm <command> --help | -h``
+
+.. _storm-lift-byverb:
+
+lift.byverb
++++++++++++
+
+The ``lift.byverb`` command lifts nodes that are connected by the specified lightweight (light) edge. By default, the command lifts the N1 nodes (i.e., the nodes on the left side of the directional light edge relationship: ``n1 -(<verb>)> n2``)
+
+.. NOTE::
+  For other commands associated with light edges, see ``edges.del`` and ``model.edge.*`` under :ref:`storm-edges` and :ref:`storm-model` respectively.
+
+**Syntax:**
+
+
+.. storm-cli:: lift.byverb --help
+
+
+.. _storm-limit:
+
+limit
+-----
+
+The ``limit`` command restricts the number of nodes returned from a given Storm query to the specified number of nodes.
+
+**Syntax:**
+
+
+.. storm-cli:: limit --help
+    
+
+**Example:**
+
+- Lift ten IP address nodes:
+
+
+.. storm-pre:: inet:ipv4 | limit 10
```

Comment:
this is a weird one - we should noodle about how to best present limit without being overly voluminous for output

Reply:
changed to apt1 ip example, and implemented limit 1 command to keep it simple